modules/model/WuerstchenModel.py

```diff
@@ -225,6 +225,7 @@ def create_pipeline(self) -> DiffusionPipeline:
                 prior_prior=self.prior_prior,
                 prior_scheduler=self.prior_noise_scheduler,
             )
+        return None
```

Comment:
Maybe we should `raise` here?

Reply:
yes, I think that's the better option. returning None would change the return type to `DiffusionPipeline | None`, which isn't what we want